sdk/messaging/azservicebus/internal/stress/shared/stress_context.go

```diff
@@ -89,7 +89,7 @@ func MustCreateStressContext(testName string) *StressContext {
 		Nano:             testRunID, // the same for now
 		ConnectionString: cs,
 		TelemetryClient:  telemetryClient,
-		statsPrinter:     newStatsPrinter(ctx, testName, 5*time.Second, telemetryClient),
+		statsPrinter:     newStatsPrinter(ctx, testName, time.Minute, telemetryClient),
```

Comment:
Nice to comment on why we want a minute.

Reply:
Oh, yeah, it's nothing special. Just long enough so I don't feel like I'm waiting forever, but not so often that @benbp  complains I'm eating up all of our diagnostics log space.

Reply:
Updated!